apps/mobile/src/features/balances/stacks/sip10-balance.tsx

```diff
@@ -0,0 +1,82 @@
+import { TokenIcon } from '@/components/widgets/tokens/token-icon';
+import {
+  useSip10AccountBalance,
+  useSip10TotalBalance,
+} from '@/queries/balance/sip10-balance.query';
+
+import { Money } from '@leather.io/models';
+import { PressableProps } from '@leather.io/ui/native';
+
+import { TokenBalance } from '../token-balance';
+
+const sip10MaxDisplay = 3;
```

Comment:
I'm adding this hard cap of 3 SIP-10 tokens until we figure out what we want to do here